packages/lib/src/interactions/DefaultInteractions.tsx

```diff
@@ -1,58 +1,80 @@
-import { merge } from 'lodash';
-
 import Pan from '../interactions/Pan';
+import type { PanProps } from '../interactions/Pan';
 import SelectToZoom from '../interactions/SelectToZoom';
+import type { SelectToZoomProps } from '../interactions/SelectToZoom';
 import XAxisZoom from '../interactions/XAxisZoom';
+import type { XAxisZoomProps } from '../interactions/XAxisZoom';
 import YAxisZoom from '../interactions/YAxisZoom';
+import type { YAxisZoomProps } from '../interactions/YAxisZoom';
 import Zoom from '../interactions/Zoom';
+import type { ZoomProps } from '../interactions/Zoom';
 import AxialSelectToZoom from './AxialSelectToZoom';
-import type { Interactions } from './models';
-import { getDefaultInteractions } from './utils';
+import type { AxialSelectToZoomProps } from './AxialSelectToZoom';
+
+export interface DefaultInteractionsConfig {
+  pan?: PanProps | false;
```

Comment:
So now we specify the props of each interaction component explicitly (not just `Interaction`). Note also that previously, we were allowing `true` to be passed, which is kind of pointless (unless I've missed something?).

Reply:
If I remember correctly, `true` was meant to be a short hand for `{}` as props (no modifier key, enabled)

Reply:
Which is pointless to use, since it's the same as not setting the property at all, right?